command/info_test.go

```diff
@@ -24,17 +24,32 @@ import (
 	"testing"
 )
 
+type infoTest struct {
+	name string
+	rawJson string
+}
+
 func TestGetInfoWorkingAsExpected(t *testing.T) {
 	client.SendRequest = func(request *http.Request, responseBody interface{}) (int, error) {
 		return http.StatusOK, nil
 	}
-	input := toJson("Test", t)
-	actualEvent := infoHandler(input)
 
-	// Issue empty because it's populated in Send request
-	expectedEvent := newInfoEvent(domain.Issue{})
-	if !reflect.DeepEqual(actualEvent, expectedEvent) {
-		t.Errorf("Expected: %v but got: %v", expectedEvent, actualEvent)
+	testCases := []infoTest {
+		{"test-normal-input", `"Test"`},
+		{"test-url-input", `"http://test123.com/Test"`},
+		{"test-slack-url-input", `"<http://test123.com/Test>"`},
```

Comment:
I would add some test cases:

https://jira.expedia.biz/browse/ELS-790
https://jira.expedia.biz/servicedesk/customer/portal/518/DEVEX-359

Just to make sure you can tolerate the additional path information and extract what you need.

Reply:
I've added all three of them. While I'm certain the first two would have worked, the last one is a very good test case and it ended up changing the logic a bit. Thanks!